de.fhdo.lemma.eclipse.ui/src/de/fhdo/lemma/eclipse/ui/ExtractLemmaModelsFromOpenApiController.xtend

```diff
@@ -0,0 +1,25 @@
+package de.fhdo.lemma.eclipse.ui
```

Comment:
It is probably not necessary to this class in the Eclipse UI plugin (cf., e.g., https://github.com/SeelabFhdo/lemma/tree/master/de.fhdo.lemma.data.avro/src/de/fhdo/lemma/data/avro/ui/to_lemma).